extension-manifest-v3/src/utils/api.js

```diff
@@ -11,7 +11,9 @@
 
 import { jwtDecode } from 'jwt-decode';
 
-const DOMAIN = 'ghostery.com';
+const DOMAIN = chrome.runtime.getManifest().debug
```

Comment:
To fix repetition problem, I suggest to change it to

```suggestion
export GHOSTERY_DOMAIN = chrome.runtime.getManifest().debug
  ? 'ghosterystage.com'
  : 'ghostery.com';
```

Then we can just use it where it is needed:

```js
import { GHOSTERY_DOMAIN } from "/utils/api";
```